frontend/src/components/ScratchList.tsx

```diff
@@ -281,9 +284,34 @@ export function ScratchItemPresetList({
     );
 }
 
+export function ScratchOwnerAvatar({ scratch }: { scratch: api.TerseScratch }) {
+    return (
+        scratch.owner &&
+        (!api.isAnonUser(scratch.owner) ? (
+            userAvatarUrl(scratch.owner) && (
```

Comment:
This could be more clearly written without ternaries: `if (scratch.owner && ...) { return <Image /> } else { return <Frog /> }`. Great idea for a component

Reply:
I just lifted this code verbatim from the Search code, but yeh that makes sense

Reply:
Can't appease the linter so gonna leave this as-is.